linux/utils/get_tpm_status.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get TPM device info in Windows guest OS
+# Return:
+#   tpm_present: the vTPM is present or not
+#   tpm_info: the dict of TPM device info
+#
+- name: "Initialize the TPM device info"
+  ansible.builtin.set_fact:
+    tpm_present: false
+    tpm_info: {}
+
+- name: "Get TPM device in guest OS"
+  ansible.builtin.shell: "ls /dev/tpm*"
+  delegate_to: "{{ vm_guest_ip }}"
+  register: tpm_dev_result
+  ignore_errors: true
+
+- name: "Display the TPM device in guest OS"
+  ansible.builtin.debug: var=tpm_dev_result
+
+- name: "Set TPM device present or not"
+  ansible.builtin.set_fact:
+    tpm_present: true
+  when:
+    - tpm_dev_result.failed is defined
+    - not tpm_dev_result.failed
+    - tpm_dev_result.stdout is defined
+    - tpm_dev_result.stdout
```

Comment:
Use `tpm_dev_result.stdout | length != 0` instead to align with the requirement in Ansible 2.19. And other `.stdout` in this task file.